src/main/scala/mesosphere/marathon/core/leadership/impl/LeadershipCoordinationActor.scala

```diff
@@ -0,0 +1,88 @@
+package mesosphere.marathon.core.leadership.impl
+
+import akka.actor.{ Actor, ActorLogging, ActorRef, Props, Stash, Status, Terminated }
+import akka.event.LoggingReceive
+import mesosphere.marathon.core.leadership.PreparationMessages
+import mesosphere.marathon.core.leadership.impl.WhenLeaderActor.{ Stopped, Stop }
+
+private[leadership] object LeadershipCoordinationActor {
+  def props(whenLeaderActors: Set[ActorRef]): Props = {
+    Props(new LeadershipCoordinationActor(whenLeaderActors))
+  }
+}
+
+private class LeadershipCoordinationActor(var whenLeaderActors: Set[ActorRef])
```

Comment:
If I understand the rule correctly, than that class should be named LeadershipCoordinatorActor.